src/game-views/Sidebar.css

```diff
@@ -1,17 +1,29 @@
 .Sidebar {
     background: #ddd;
-    border-left: 3px solid #282c34;
     display: flex;
     flex-direction: column;
     grid-area: sidebar;
+    overflow-x: hidden;
+}
+
+@media (min-width: 950px) {
+    .Sidebar {
+        border-left: 3px solid #282c34;
+    }
 }
 
 @media (min-width: 1750px) {
     .Sidebar {
         flex-direction: row;
     }
     .Sidebar-players {
-        min-width: 530px;
+        min-width: 510px;
+    }
+}
+
+@media (max-width: 444px) {
```

Comment:
when do you decide to max-width vs min-width? is there anything special about 444?